Added find_and_replace.py to Automation

guillearch opened the pull request:
# Description

The script **find_and_replace.py** finds and replaces all ocurrences of a given string with another given string in all the files of a given directory.

## Type of change

- [X] New Script

guillearch committed:
Added find_and_replace.py to Automation

nishantnahata reviewed (changes requested):
An automated program should handle it's errors by itself:-
1. Add error handling for source and destination, if the directory doesn't exist.
2. Add error handling for files which are not readable and throws some exception.
3. Use ```os.path.join()``` instead of ```+``` to join source and destination to files.

Some additional improvements that can be made:
1. Give user an option to recursively go through directories for replacement.
2. If possible, give an option to pass all these parameters in command line arguments.

ionicc reviewed (changes requested):
As @nishantcoder97 said.
Please make sure to always use `os.path.join()` instead of direct concatenation 😄

guillearch committed:
Added error handling for source and destination

guillearch committed:
Replaced direct concatenations with os.path.join()

guillearch commented:
I added error handling for source and destination and replaced direct concatenations with os.path.join().

I'd be happy to work on the additional improvements suggested by @nishantcoder97 when I have more time.
